### PR TITLE
Update circle.yml to use node 18

### PR DESCRIPTION
**Pull request by winniehell**
uses current LTS of node (18) for CircleCI

this may unblock https://github.com/posva/vuex-mock-store/pull/186 and https://github.com/posva/vuex-mock-store/pull/197

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:10
+      - image: circleci/node:18
```
Comment (winniehell) [inline circle.yml:11]:
```suggestion
      - image: cimg/node:lts
```

see https://circleci.com/developer/images/image/cimg/node

**Comment by winniehell** [comment]
@posva can you please merge?

File: circle.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:10
+      - image: cimg/node:lts
```
Comment (winniehell) [inline circle.yml:11]: https://circleci.com/developer/images/image/cimg/node

**Comment by winniehell** [comment]
superseded by https://github.com/posva/vuex-mock-store/pull/202

**Comment by posva** [comment]
Thanks a lot!

**Pull request closed without merging**